src/System.Runtime.WindowsRuntime/pkg/build/net461/System.Runtime.WindowsRuntime.targets

```diff
@@ -0,0 +1,20 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Choose>
+    <When Condition="'$(DoNotReferenceWinRT)' != 'true'">
+      <PropertyGroup>
+        <!-- include the reference to System.Runtime -->
+        <_HasReferenceToSystemRuntime>true</_HasReferenceToSystemRuntime>
```

Comment:
@ericstj do we need this property here? I thought that on netsntadard2.0 the DependsOnNETStandard property should be enough.

Reply:
@joperezr that may be possible, I will give it a try.  Originally we were using _HasReferenceToSystemRuntime and for netstandard2.0 it was not sufficient... I did not try removing that for this scenario.

Reply:
Good suggestion, I have removed this variable from the net461 use (eg. netstandard2.0)